recipes/gau2grid/bld.bat

```diff
@@ -0,0 +1,33 @@
+cmake -G"Ninja" ^
+      -H%SRC_DIR% ^
+      -Bbuild ^
+      -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+      -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+      -DCMAKE_INSTALL_LIBDIR="%LIBRARY_LIB%" ^
+      -DCMAKE_INSTALL_INCLUDEDIR="%LIBRARY_INC%" ^
+      -DCMAKE_INSTALL_BINDIR="%LIBRARY_BIN%" ^
+      -DCMAKE_INSTALL_DATADIR="%LIBRARY_PREFIX%" ^
+      -DPYMOD_INSTALL_LIBDIR="/../../Lib/site-packages" ^
```

Comment:
```suggestion
      -DPYMOD_INSTALL_LIBDIR="%SP_DIR%" ^
```

Reply:
Thanks for the apparently sensible suggestion, but no, `PYMOD_INSTALL_LIBDIR` is a string appended to `CMAKE_INSTALL_LIBDIR` to build a full path, so the full path `SP_DIR` won't work.